src/components/BookmarkPage.js

```diff
@@ -3,16 +3,47 @@ import RestaurantBox from "./RestaurantBox";
 import { ReactComponent as Find } from "../icons/find.svg";
 import { Link } from "react-router-dom";
 
-export default function BookmarkPage() {
+export default function BookmarkPage({ restaurantData }) {
+  const bookmarkedRestaurants = restaurantData.filter(
+    (savedRestaurants) => savedRestaurants.isBookmarked === true
```

Comment:
The function you pass to the `.filter` method is receiving each `restaurant`, not all the `savedRestaurants`. I've call this `restaurant` or `savedRestaurant`

Exactly how you are doing with the `map` in line 12